de/taktisch.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "uuid": "c19195e4-c444-4e12-bade-f0d011436440",
+    "title": "Truppführer",
+    "abbreviation": "TF",
+    "category": {
+      "uuid": "605f7761-73df-4b41-a992-19a765f4bf49",
+      "title": "Taktisch"
+    },
+    "includes": [],
+    "included_by": []
+  },
+  {
+    "uuid": "9b9f0c05-aa5b-4f4e-8afd-7b4fdffd314a",
+    "title": "Gruppenführer",
+    "abbreviation": "GF",
+    "category": {
+      "uuid": "605f7761-73df-4b41-a992-19a765f4bf49",
+      "title": "Taktisch"
+    },
+    "includes": [
+      "c19195e4-c444-4e12-bade-f0d011436440"
+    ],
+    "included_by": []
```

Comment:
with the other files we put values in `included_by` too. I think we should do that here as well?!

Reply:
why do we need that/where do we use that? at least importing worked flawlessly including all inclusions

Reply:
we don't need it, true. I'm fine with leaving it out...